src/components/Navigation/Navigation.scss

```diff
@@ -348,11 +348,11 @@ $secondary-item-font-size: rem(15px);
   padding-left: spacing();
 
   .Text {
-    font-size: font-size(subheading);
+    font-size: var(--fs-1);
```

Comment:
Glad to see we settled on token scales for `font-size` and `line-height`!

**suggestion(non-blocking):** Wonder if we should expand these abbreviations to match the `--p-font-weight` tokens @lgriffee [just introduced](https://github.com/Shopify/polaris-react/blob/7d97674ab6434618a8d1d649022b5c1baefcdf40/documentation/Color%20system.md?plain=1#L264-L267)? I know this is still WIP and the scope of this work is to get 100% token coverage, but I feel like we should have consistency in our typography tokens.

```css
p {
  font-size: var(--p-fs-1);
  font-weight: var(--p-font-weight-regular)
}
```
This doesn't feel very intuitive

Reply:
I don't feel too strongly either way. Now is the time to play around with this and it will be easy enough to change through the token generator. Would it be worth creating a separate issue to align on naming convention?

Reply:
I don't think a separate issue is necessary personally. If anything I would just have a quick sync with @lgriffee to settle on a convention for v8. Right now it is clear two developers worked on the typography vars. I'm leaning towards expanding since `fs` isn't clear to me. If I was looking at a collection of `:root` properties does `fs` mean: `flex-shrink`, `font-style`, `font-stretch`, `font-synthesis`, `font-size`, `font-size-adjustment`, etc...?